### DISCUSSION TITLE
Official C++ Build For Godot 4.3

### DISCUSSION THREAD

**Discussion by VDev7** [godotengine/godot-proposals#8997] [Scripting]
Since Godot Is Written Using C++ Isn't It Make Sense If There Is Official C++ Build At Download Section So We Doesn't Need To Use GDExtension For Godot Real Language, I Know That It Would Be Hard Maintaining 3 Language At The Same Time As Its Already Hard For 2 Language

**Comment by theraot** [comment]
What Would Be Different About This Hypothetical Official C++ Build Compared To The Standard Versions?

**Comment by theraot** [comment]
Are you asking for something similar to this proposal: https://github.com/godotengine/godot-proposals/issues/7757 ?

**Reply by VDev7** [reply]
Yes
